toxcore/tox.h

```diff
@@ -3303,6 +3303,16 @@ uint32_t tox_group_max_part_length(void);
  */
 #define TOX_GROUP_MAX_MESSAGE_LENGTH    1368
 
+/**
+ * Maximum length of a group custom lossy packet.
+ */
+#define TOX_GROUP_MAX_CUSTOM_LOSSY_PACKET_LENGTH 500
```

Comment:
500 is about largest size of a UDP packet that can safely traverse almost any network networks. See [this thread ](https://stackoverflow.com/questions/1098897/what-is-the-largest-safe-udp-packet-size-on-the-internet)for further explanation.

Reply:
Very interesting, might be worth linking that as the source for this value, because otherwise `500` is very magic.

Reply:
I still think that for LOSSY 500 is an arbitrary low limit. I am full on board for LOSSLESS however.

Reply:
500 seems just a random number, since toaxv is using a much higer limit (about 13xx bytes) for lossy packets.
so if toxav is ok, why should groups not be?

Reply:
The number is not "random" and I have already explained why that value was chosen. I also added a link to said explanation in a comment. If older parts of toxcore are wrong, it does not logically follow that we should repeat those mistakes in new code, which is what your argument is tantamount to.

Reply:
also toxcore itsself uses UDP and packets of about 1373 bytes.